vision/app.py

```diff
@@ -80,8 +80,7 @@ def run_image(self):
 			cv2.destroyAllWindows()
 
 	def run_video(self):
-		cameraFront = WebcamVideoStream(src=self.source[1]).start()
-		cameraRear = WebcamVideoStream(src=self.source[0]).start()
+		camera = WebcamVideoStream(src=self.source[0]).start()
```

Comment:
`src=self.source` and change `sources` to `source` in [utils.py](https://github.com/SouthEugeneRoboticsTeam/vision/blob/master/vision/utils.py#L17), along with changing the type to `int`, the default to `0`, and the help text to `video sources (default=0)`.